rules/camunda-cloud/no-loop.js

```diff
@@ -38,16 +38,17 @@ module.exports = skipInNonExecutableProcess(function() {
       return;
     }
 
-    const error = getFlowElements(node)
+    // Create subgrapg of nodes that can be part of an infinite loop
```

Comment:
Change to `subgraph`

Reply:
Done in [2dd0b78](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/165/commits/2dd0b782a8e4bdd7fa22c3140aaa84d8f867e3f7)